### ISSUE TITLE
AutoLogging - Remote resource DataRetentionRulesResourceImpl is not configured for autologging.

### ISSUE THREAD

**Issue by stroomdev10** [gchq/stroom#3278]

**Comment by at055612** [comment]
the cancel query rest endpoint is now manually logged, e.g

```
  <EventDetail>                                                                                                                                                                                                                                                                 
    <TypeId>DataRetentionRulesResourceImpl.cancelQuery</TypeId>                                                                                                                                                                                                                 
    <Description>Cancelling data retention delete summary with query id 8316544B-F98C-41CE-B585-1E41A1C0978A</Description>                                                                                                                                                      
    <Delete>                                                                                                                                                                                                                                                                    
      <Criteria>                                                                                                                                                                                                                                                                
        <Query>                                                                                                                                                                                                                                                                 
          <Advanced>                                                                                                                                                                                                                                                            
            <Term>                                                                                                                                                                                                                                                              
              <Name>queryId</Name>                                                                                                                                                                                                                                              
              <Condition>Equals</Condition>                                                                                                                                                                                                                                     
              <Value>8316544B-F98C-41CE-B585-1E41A1C0978A</Value>                                                                                                                                                                                                               
            </Term>                                                                                                                                                                                                                                                             
          </Advanced>                                                                                                                                                                                                                                                           
        </Query>                                                                                                                                                                                                                                                                
      </Criteria>                                                                                                                                                                                                                                                               
    </Delete>                                                                                                                                                                                                                                                                   
  </EventDetail>    
```

**Comment by at055612** [comment]
Fixed in >7.0.24
